app/src/main/kotlin/com/softteco/template/ui/AppContent.kt

```diff
@@ -32,17 +40,24 @@ fun AppContent(
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
         RequestNotificationPermissionDialog()
     }
-    Scaffold(
-        modifier = modifier.fillMaxSize(),
-        containerColor = MaterialTheme.colorScheme.background,
-        bottomBar = {
-            AppBottomBar(navController = navController)
+
+    ProvideSnackbarController(
```

Comment:
It's a good solution to use  CompositionLocalProvider, but it would be better use it here directly, to have possibility to add new items to CompositionLocalProvider. 
    
```
CompositionLocalProvider(
        LocalSnackbarController provides SnackbarController(
            snackbarHostState = snackbarHostState,
            coroutineScope = coroutineScope
        )
    ) {
        Scaffold(...

Reply:
it's also in AppContent
```
val LocalSnackbarController = staticCompositionLocalOf<SnackbarController> {
    error("No SnackbarController provided.")
}
```

Reply:
fixed